src/features/hotspots/checklist/HotspotChecklistItem.tsx

```diff
@@ -74,68 +78,49 @@ type Props = {
   title: string
   description: string
   complete?: boolean
-  completeText?: string
   showAuto?: boolean
   autoText?: string
-  background?: 1 | 2 | 3 | 4
+  itemKey: string
   isAndroid: boolean
+  percentComplete: number
+  index: number
+  totalCount: number
 }
 const HotspotChecklistItem = ({
   title,
   description,
   complete,
-  completeText,
   showAuto,
   autoText,
-  background,
   isAndroid,
+  percentComplete,
+  itemKey,
+  index,
+  totalCount,
 }: Props) => {
   const { t } = useTranslation()
   const colors = useColors()
-  const spacing = useSpacing()
-  const textColor = complete ? 'greenDarkText' : 'white'
-  const backgroundColor = complete ? 'greenChecklist' : 'purpleMain'
-  const backgroundColorHex = complete
-    ? colors.greenChecklist
-    : colors.purpleMain
 
-  const Background = useCallback(() => {
-    switch (background) {
+  const icon = useMemo(() => {
+    const color = complete ? colors.purpleMain : '#C2C5E4'
```

Comment:
Is there something in the theme close to #C2C5E4?

Reply:
No but I added a purpleGrayLight